src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -251,8 +251,11 @@ bool mpuCheckDataReady(gyroDev_t* gyro)
 static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro)
 {
     UNUSED(gyro); // since there are FCs which have gyro on I2C but other devices on SPI
+
 #ifdef USE_GYRO_SPI_MPU6000
+#ifdef MPU6000_SPI_INSTANCE
     spiBusSetInstance(&gyro->bus, MPU6000_SPI_INSTANCE);
+#endif
 #ifdef MPU6000_CS_PIN
     gyro->bus.spi.csnPin = gyro->bus.spi.csnPin == IO_NONE ? IOGetByTag(IO_TAG(MPU6000_CS_PIN)) : gyro->bus.spi.csnPin;
```

Comment:
(`IO_NONE` is for IO_t, this code should use simple boolean conversion (preferred) or `IO_TAG_NONE`